### PR TITLE
buttons completed

### PR DESCRIPTION
**Pull request by MayurChawla**
I have created buttons with different style, i have also created some custom buttons
buttons
.

### CODE REVIEW

**Comment by aurobindoGupta** [comment]
nice work

**Review by aurobindoGupta — APPROVED** [review]

File: Buttons/Buttons.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    <link
+        href="https://fonts.googleapis.com/css?family=Material+Icons|Material+Icons+Outlined|Material+Icons+Two+Tone|Material+Icons+Round|Material+Icons+Sharp"
+        rel="stylesheet">
+    <link rel="stylesheet" href="buttonStyle.css">
+</head>
+
+<body>
+    <div class="button-container">
+        <section class="basicButtons">
+            <div class="title">Basic</div>
```
Comment (junaidshaikh-js) [inline Buttons/Buttons.html:18]: You can use heading tag for title

File: Buttons/buttonStyle.css [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+    --btn-primary : #007bff;
+    --btn-primary_hover : #b1d7ff;
+    --btn-success : #00bc49;
+    --btn-success_hover : rgb(189, 255, 189);
+    --btn-warning : #ffc107;
+    --btn-warning_hover : #fde9ae;
+    --btn-original :rgb(239, 239, 239);
+    --btn-link : #4B1F0E;
+}
+* {
```
Comment (junaidshaikh-js) [inline Buttons/buttonStyle.css:20]: can be moved to reset.css file if you have one

File: Buttons/buttonStyle.css [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+}
+.flatButtons .warn {
+    background-color: var(--btn-warning);
+    color: white;
+}
+.flatButtons .disabled {
+    opacity: 0.5;
+}
+.flatButtons .link {
+    background-color: white;
```
Comment (junaidshaikh-js) [inline Buttons/buttonStyle.css:198]: A lot of time white is getting repeated, you can use variable for that

**Review by junaidshaikh-js — COMMENTED** [review]
Good Work!

File: Buttons/buttonStyle.css [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+    opacity: 0.5;
+}
+.iconButtons .link:hover {
+    background-color: var(--btn-original);
+}
+
+/* Floating Action Buttons */
+.fabButtons .btn {
+    background-color: var(--primary-color);
+    color: white;
```
Comment (aurobindoGupta) [inline Buttons/buttonStyle.css:238]: var()

File: Buttons/buttonStyle.css [hunk 5]
```diff
@@ -0,0 +1,346 @@
...
+}
+.miniFabButtons .link {
+    background-color: var(--btn-link);
+    color: white;
+}
+
+/* Custom FAB */
+.customButtons .btn {
+    background-color: var(--primary-color);
+    color: white;
```
Comment (aurobindoGupta) [inline Buttons/buttonStyle.css:312]: var()

File: Buttons/buttonStyle.css [hunk 6]
```diff
@@ -0,0 +1,346 @@
...
+    padding: 16px;
```
Comment (aurobindoGupta) [inline Buttons/buttonStyle.css:313]: rem

File: Buttons/buttonStyle.css [hunk 7]
```diff
@@ -0,0 +1,346 @@
...
+    border-radius: 50%;
+}
+.customButtons .primary {
+    border-radius: 24px;
```
Comment (aurobindoGupta) [inline Buttons/buttonStyle.css:317]: rem

**Review by aurobindoGupta — APPROVED** [review]

**Pull request merged**